lifetimes/utils.py

```diff
@@ -48,7 +48,7 @@ def to_period(d):
 
     # create calibration dataset
     calibration_transactions = transactions.ix[transactions[datetime_col] <= calibration_period_end]
-    calibration_summary_data = summary_data_from_transaction_data(calibration_transactions, customer_id_col, datetime_col,
+    calibration_summary_data = summary_data_from_transaction_data(calibration_transactions, customer_id_col, datetime_col,monetary_value_col,
```

Comment:
Actually,when I tried implementing the previous function, it was throwing some error.